apps/cognito-functions/src/define-auth-challenge-handler.ts

```diff
@@ -0,0 +1,52 @@
+import { DefineAuthChallengeTriggerEvent } from 'aws-lambda';
+
+export const makeHandler =
+  () =>
+  async (
+    event: DefineAuthChallengeTriggerEvent
+  ): Promise<DefineAuthChallengeTriggerEvent> => {
+    const { session } = event.request;
+
+    if (session.length === 1 && session[0].challengeName === 'SRP_A') {
+      // SRP_A is the first challenge, this will be implemented by cognito. Set
+      // next challenge as PASSWORD_VERIFIER.
+      const response = {
+        challengeName: 'PASSWORD_VERIFIER',
+        failAuthentication: false,
+        issueTokens: false,
+      };
+      return { ...event, response };
+    } else if (
+      session.length === 2 &&
+      session[1].challengeName === 'PASSWORD_VERIFIER' &&
+      session[1].challengeResult == true
```

Comment:
```suggestion
      session[1].challengeResult
```
this should work too